components/dashboard/src/projects/Projects.tsx

```diff
@@ -90,23 +97,16 @@ export default function () {
                         <div className="py-4">
                             <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16" width="16" height="16"><path fill="#A8A29E" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" /></svg>
                         </div>
-                        <input type="search" placeholder="Search Projects" onChange={(e) => onSearchProjects(e.target.value)} />
+                        <input type="search" placeholder="Search Projects" onChange={e => setSearchFilter(e.target.value)} />
```

Comment:
**issue**: It seems that search is triggered after typing at least two characters, right? Could we use the same interaction as elsewhere where search is triggered from typing one character? For example, see workspaces search input.

Reply:
> It seems that search is triggered after typing at least two characters, right? 

that's not the case. 

though, probably it's better to not include the clone URL to search in.

Reply:
Oh! Shouldn't both project disappear when typing the first character in the screencast below?

https://user-images.githubusercontent.com/120486/130454350-11cf4e1f-48de-4bd4-b594-9880468f44eb.mov

Reply:
> Oh! Shouldn't both projects disappear when typing the first character in the screencast below?

no, because "a" is included in the clone URL.

Reply:
updated. resolved.

Reply:
Ah, ok! Got it. Didn't realize we include the whole clone URL. Thanks, @AlexTugarev! 🧙